Themis/Views/Exercises/ExerciseView.swift

```diff
@@ -50,6 +65,11 @@ struct ExerciseView: View {
             )
         }
         .navigationTitle(exercise.title ?? "")
+        .toolbar {
+            ToolbarItem(placement: .automatic) {
+                Text("")
```

Comment:
What is this for?

Reply:
It was used for testing purposes, must've slipped through. Removed in the latest commit.